### DISCUSSION TITLE
Discord link not working

### DISCUSSION THREAD

**Discussion by fabi200123** [akuity/kargo#2999] [Q&A]
I wanted to join the community and ask some questions about how to configure Kargo with Dex, but apparently the link to join discord might be expired

**Comment by krancour** [comment]
It seems to be working fine for me.

@christianh814 as the resident Discord guru, would you mind looking into this when you have a chance?

**Comment by christianh814** [comment]
@fabi200123 I just tested it and it is working as intended

Can you try going to https://akuity.community as well

**Reply by fabi200123** [reply]
The link works, thank you!
But apparently the discord link from github didn't work for me

**Reply by krancour** [reply]
I did encounter some kind of redirect loop when using the link from my phone. I didn't have the same difficulty with my desktop. It's possible this has something to do with Discord being installed on my desktop and not my phone? Conjecture. 🤷‍♂️ But I don't think we're doing anything out of the ordinary, so if we observe such things, I doubt it's unique to us.

**Reply by krancour** [reply]
Closing this since it's confirmed to be working.
